### PR TITLE
Sergio/teams model

### PR DESCRIPTION
**Pull request by sgarnica1**

### CODE REVIEW

File: src/models/question.model.js [hunk 1]
```diff
@@ -119,8 +119,8 @@ class Question {
     return res;
   }
 
-  delete() {
-    // TODO
+  async delete() {
+    return await db.execute(`DELETE FROM question WHERE id = ?`, [this.id]);
```
Comment (DanielSebasCM) [inline src/models/question.model.js:123]: Questions should be deleted, they should be soft deleted (active = false)
```suggestion
    return await db.execute(`UPDATE question SET active = 0 WHERE if = ?`, [this.id]);
    
```

File: src/testing/team.test.js
[the remark this answers is not in the document]
Comment (sgarnica1) [reply src/testing/team.test.js:1]: Sorry, forgot about that

File: src/testing/question.test.js
[the remark this answers is not in the document]
Comment (sgarnica1) [reply src/testing/question.test.js:1]: I think it is better to create a testing DB, so methods shouldn't be modified for production

File: src/models/question.model.js (hunk 1 above)
Comment (sgarnica1) [reply src/models/question.model.js:123]: It is better this way

File: src/testing/question.test.js [hunk 2]
```diff
@@ -111,6 +111,9 @@ test("question inserted successfully", async () => {
   expect(mockQuestion.description).toEqual(createdQuestion.description);
   expect(mockQuestion.type).toEqual(createdQuestion.type);
   expect(createdQuestion.active).toEqual(1);
+
+  // delete question
+  await mockQuestion.delete();
```
Comment (DanielSebasCM) [inline src/testing/question.test.js:116]:
```suggestion
```

**Review by DanielSebasCM — APPROVED** [review]

**Pull request merged**